pkg/testgridanalysis/testgridconversion/to_raw_data.go

```diff
@@ -29,14 +31,19 @@ type ProcessingOptions struct {
 	NumDays              int
 }
 
+type lookback struct {
+	startCol int
+	endCol   int
+}
```

Comment:
was this driven by the linter?  I agree there's a point at which combining multiple args into a struct makes sense.  i'm not positive that "2" is the point at which it becomes important though.

is it specifically tied to the use as a return value to avoid multiple return values?

Reply:
Yes it was. I agree with your point. It identified this and `utils.ComputeFailureGroupStats` as potential hotspots. In the latter case, I think fixing it makes sense. I'll revert the change for the `lookback` struct and disable that particular linter.